timeout_test.go

```diff
@@ -38,6 +38,14 @@ func (ct *ctxTable) ReadRows(
 	return nil
 }
 
+func (ct *ctxTable) ReadRow(
+	ctx context.Context,
+	row string,
+	opts ...bigtable.ReadOption,
+) (bigtable.Row, error) {
+	return nil, nil
```

Comment:
This is totally fine, but if you don't actually use the implementation here, you can also embed the interface directly in the struct for convenience.

Reply:
I think that became unnecessary when I removed the other tests. I'll remove it now.

Reply:
Nevermind, I'll just leave it as-is for now.